go.mod

```diff
@@ -176,10 +148,13 @@ require (
 	google.golang.org/genproto v0.0.0-20220628213854-d9e0b6570c03 // indirect
 	google.golang.org/grpc v1.48.0 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 	gopkg.in/ini.v1 v1.66.6 // indirect
+	gopkg.in/natefinch/npipe.v2 v2.0.0-20160621034901-c1b8fa8bdcce // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
-	nhooyr.io/websocket v1.8.6 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+require github.com/ComposableFi/go-substrate-rpc-client/v4 v4.0.1-0.20220815083349-ea2c4ccf9f2e
```

Comment:
This is a fork of [centrifuge](https://github.com/centrifuge/go-substrate-rpc-client/) repo, I made it separate from other packages because we need to replace it with the upstream version after merging to their repo. however, i can move it next to the rest.

Reply:
Done